.github/dependabot.yml

```diff
@@ -0,0 +1,257 @@
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/local_auth/local_auth_android/example/android/app"
+    commit-message:
+      prefix: "[local_auth_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/local_auth/local_auth/example/android/app"
+    commit-message:
+      prefix: "[local_auth]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/path_provider/path_provider/example/android/app"
+    commit-message:
+      prefix: "[path_provider]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/path_provider/path_provider_android/example/android/app"
+    commit-message:
+      prefix: "[path_provider_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/quick_actions/quick_actions_android/android"
+    commit-message:
+      prefix: "[quick_actions_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/quick_actions/quick_actions_android/example/android/app"
+    commit-message:
+      prefix: "[quick_actions_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/quick_actions/quick_actions/example/android/app"
+    commit-message:
+      prefix: "[quick_actions]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/shared_preferences/shared_preferences/example/android/app"
+    commit-message:
+      prefix: "[shared_preferences]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/shared_preferences/shared_preferences_android/example/android/app"
+    commit-message:
+      prefix: "[shared_preferences_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/url_launcher/url_launcher_android/android"
+    commit-message:
+      prefix: "[url_launcher_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/url_launcher/url_launcher_android/example/android/app"
+    commit-message:
+      prefix: "[url_launcher_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/url_launcher/url_launcher/example/android/app"
+    commit-message:
+      prefix: "[url_launcher]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
```

Comment:
The app-facing package's example is missing.

Reply:
url_launcher doesn't have an app-facing package. https://github.com/flutter/plugins/tree/main/packages/url_launcher/url_launcher